pkg/agent/util/net_linux.go

```diff
@@ -378,6 +384,17 @@ func renameHostInterface(oriName string, newName string) error {
 	return nil
 }
 
+func removeInterfaceAltName(name string, altName string) error {
```

Comment:
Shell we add the altname back in restore case?

Reply:
No, this altname is conflict with original uplink name, thus no need to restore it.